scripts/libuvc_installation.sh

```diff
@@ -40,6 +40,7 @@ cd ./librealsense-master
 
 echo Install udev-rules
 sudo cp config/99-realsense-libusb.rules /etc/udev/rules.d/ 
+sudo cp config/99-realsense-d4xx-mipi-dfu.rules /etc/udev/rules.d/
```

Comment:
If this script is applied on linux machine, there will be no such path as /sys/class/d4xx-class/d4xx-dfu... - will this lead to a failure?

Reply:
no, i checked that

Reply:
great- thanks!

Reply:
This PR should be based on development branch, and aimed to development branch - please change that